wfe/wfe.go

```diff
@@ -637,6 +637,13 @@ func (wfe *WebFrontEndImpl) NewRegistration(ctx context.Context, logEvent *web.R
 		return
 	}
 
+	if !features.Enabled(features.AccountCreation) {
+		wfe.sendError(response, logEvent, probs.Unauthorized("Account creation on ACMEv1 is disabled. "+
+			"Please upgrade your ACME client to a version that supports ACMEv2. "+
```

Comment:
```suggestion
			"Please upgrade your ACME client to a version that supports ACMEv2 / RFC 8555. "+
```

I think there (rightly) may be ACME clients that advertise themselves as RFC 8555 compatible and don't use our invented "ACME v2"